crates/ide/src/runnables.rs

```diff
@@ -1503,23 +1301,7 @@ fn test_foo1() {}
                             name: "test_foo1",
                             kind: Function,
                         },
-                        kind: Test {
-                            test_id: Path(
-                                "test_foo1",
-                            ),
-                            attr: TestAttr {
-                                ignore: false,
-                            },
-                        },
-                        cfg: Some(
```

Comment:
This is interesting data (as long as its `Some`), same for when `use_name_in_title` is `true`. Would be good to render those conditionally (or otherwise keep them in the output in some way)

Reply:
Sure, gonna add that.